www/js/lib/zimArchive.js

```diff
@@ -109,14 +109,62 @@ define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
      */
 
     /**
+     * Look for titles starting with the given prefix.
+     * For now, ZIM titles are case sensitive.
+     * So, as workaround, we try several variants of the prefix to find more results.
+     * This should be enhanced when the ZIM format will be modified to store normalized titles
+     * See https://github.com/mossroy/evopedia-html5/issues/117
      * 
      * @param {String} prefix
      * @param {Integer} resultSize
-     * @param {type} callback
-     * @returns {callbackTitleList}
+     * @param {callbackTitleList} callback
      */
     ZIMArchive.prototype.findTitlesWithPrefix = function(prefix, resultSize, callback) {
         var that = this;
+        that.findTitlesWithPrefixCaseSensitive(prefix, resultSize, function(titles) {
+            if (titles.length < resultSize) {
```

Comment:
It's a matter of style, but I think it is better readable to reverse the cases:
```
if (titles.length >= resultSize) {
  callback(titles);
  return;
}
...
```
(and have to explicit else case to reduce the indentation).